app/models/repo.rb

```diff
@@ -161,6 +161,7 @@ def create_branch!(branch_name)
   end
 
   def git_service
+    git_fetch
```

Comment:
Drive by comment...

Would an alternative for this just use a shared volume between worker pods so that we aren't doing a bunch of duplicate clones per worker?  Just thinking that cloning `ManageIQ` for every worker container seems... expensive.

But I could also be full of it, and that also doesn't really invalidate this change, just questioning the justification for it and if it could be solved in a different way.